public/sites/default/all.services.yml

```diff
@@ -7,3 +7,9 @@ parameters:
     - 'avustukset.dev.hel.ninja'
     - 'avustukset.stage.hel.ninja'
 
+services:
+  resilient_logger.service:
```

Comment:
When possible, service name should the class name (`ResilientLogger\ResilientLogger`). This makes autowiring the service is easier and helps IDEs type check.